### PR TITLE
Support configurable PodMetadata and scale out EACRuntime in parallel

### PR DESCRIPTION
**Pull request by TrafalgarZZZ**
### Ⅰ. Describe what this PR does
Scale out EACRuntime in parallel. This PR sets `podManagementPolicy` of EACRuntime's worker statefulset to `Parallel` to speed up the scaling process.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: charts/eac/values.yaml [hunk 1]
```diff
@@ -71,6 +75,8 @@ fuse:
   #      memory: "4G"
 
 master:
+  labels: {}
```
Comment (cheyang) [inline charts/eac/values.yaml:78]: Please also update chart version and changelog.

Comment (TrafalgarZZZ) [reply charts/eac/values.yaml:78]: Done

**Pull request merged**